e2e/cypress/integration/integrations/incoming_webhook/long_url_embedded_image_spec.js

```diff
@@ -0,0 +1,86 @@
+
+function getPayload(testChannel) {
+    return {
+        channel: testChannel.name,
+        text: `Hey attachments ![graph](${url})${text}`,
+    };
+}
```

Comment:
Same here. Other text values don't need to be exactly as the test data suggest. So, this can simply,
```javascript
function getPayload(channel, url) {
    const text = `Hey attachments ![graph](${url}). ${'Lorem ipsum dolor '.repeat(240)}.`;

    return {
        channel: channel.name,
        text,
    };
}
```